docs/Home.md

```diff
@@ -1,3 +1,13 @@
-Welcome to the android-templates wiki! 
+Welcome to the android-templates wiki!
 
-More coming soon...
\ No newline at end of file
+The purpose of our android-templates repository is to quickly start a new project with all the **must-have** requirements. Other **nice-to-have** features need to be considered as an optional setting when starting a new project.
+
+Planning to add a new feature? Ask yourself the following questions to define if it's a **must-have** requirement:
+
+- Is the feature frequently used in other projects? (Ex: Glide, Retrofit, ...)
```

Comment:
frequently used or having the side setup that is usually verbose to replicate is also considerable.

Reply:
>  having the side setup that is usually verbose to replicate

Could you give an example of this, please? 🙇

Reply:
for example OkHttp we usually have to configure:
- Debug level
- Timeout for read/write
- Interceptor
...

Reply:
Then I would say they are frequently used. Debug level, time-outs, and interceptors are frequently used in other projects. So I think we keep it like it is, to keep it minimized. Instead, I could mention it as one of the examples for a must-have feature (with some more explanation, like we did for Lottie). What do you think? 🙏

Reply:
well...it's also good if you can specify on the setup as well; frequently used setups could be different from frequently used lib - as a lib could have many features that we don't use, so my point here is also to welcome people open PR to make configuration changes that we can make used of too...not only about adding a new lib should be considered as a must-have contribution.

Reply:
What about if we rephrase it like this?

>  Is the feature frequently used/configured in other projects?

Reply:
Or we need to avoid using "feature" as it might not clearly refer to configurations as you mentioned 🤔